Makefile

```diff
@@ -71,10 +71,13 @@ copy-files:
 	install -m 600 srv/salt/ceph/salt-api/files/sharedsecret.conf.j2 $(DESTDIR)/etc/salt/master.d/sharedsecret.conf
 	# qa
 	install -d -m 755 $(DESTDIR)/usr/lib/deepsea/qa/common
+	install -d -m 755 $(DESTDIR)/usr/lib/deepsea/qa/osd-config
```

Comment:
this line is also not needed because `install -d` automatically creates intermediate directories - i.e. `qa/osd-config` will get created in the next line